src/spatio_temporal_voxel_grid.cpp

```diff
@@ -174,8 +174,7 @@ void SpatioTemporalVoxelGrid::TemporalClearAndGenerateCostmap(                \
         }
         else
         {
-          const double updated_mark = cit_grid.getValue() - \
-            (decay_shift - accel_decay_shift);
+          const double updated_mark = cit_grid.getValue() - decay_acceleration;
```

Comment:
decay_acceleration -> accelerated_duration or something similar. acceleration and decay are both in the same vein

Reply:
How about just `acceleration`?

Reply:
see comment above

Reply:
see comment above

Reply:
see comment above

Reply:
see comment above

Reply:
see comment above